recipe/build.sh

```diff
@@ -100,15 +120,20 @@ if [[ "${CI}" == "github_actions" ]]; then
     # cirun-openstack-gpu-2xlarge, which has 32GB RAM, 8 CPUs
     export MAX_JOBS=4
 else
-    export MAX_JOBS=${CPU_COUNT}
+    # Leave a spare core for other tasks. This may need to be reduced further
+    # if we get out of memory errors.
+    export MAX_JOBS=$((CPU_COUNT > 1 ? CPU_COUNT - 1 : 1))
```

Comment:
What happens if `CPU_COUNT` is undefined or empty string?

Reply:
What CI provider do things run on for anaconda? I think it might need to become specific to that, and not in the overall `else:` clause here. Our OSX builds on azure work are successful withing the 6h limit, but seem to fail with this PR. This is a likely culprit.

Reply:
It's AWS. Happy to just leave it as-is. I think originally we had it maxed at four cores and then increased it to CPU_COUNT-1 because of common practice and a desire to speed things up rather than concrete technical reasons. We can leave this as-is and if we have issues our end solve them later

Reply:
I've fixed our case in 6074128b18c469590ff29fb9485f4539daf86386, so I have no objection to doing what helps you in the `else:` branch.

Reply:
have made this a default in the else branch

Reply:
Note that [`CPU_COUNT` is a passthrough environment variable]( https://github.com/conda/conda-build/blob/8df6493565dae517165754bd75d7d74de1522dc3/conda_build/environ.py#L794 )

So one can set this in their CI scripts to the intended value and conda-build will use it. This is [what conda-forge does]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/d8b11fd85622f4d332cd099d91e8fa12562c9be6/recipe/run_conda_forge_build_setup_linux#L20-L28 )

Instead of adding this to the build recipe, would recommend doing this in Anaconda's CI scripts

Reply:
`CPU_COUNT` isn't being set here. If it's set (somewhere else, like in CI as you suggest) then it'll be used to set `MAX_JOBS`. If it's not set, this expression will evaluate to 1